packages/terra-menu/src/_MenuUtils.js

```diff
@@ -0,0 +1,57 @@
+const getPopupHeight = (contentHeight) => {
+  if (contentHeight <= 40) {
```

Comment:
Sorry! I was thinking Math.min here. It looks like you want to return a height that is a multiple of 40 up to 880, correct? If so, I think the min function would work here.